ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -382,7 +402,7 @@ calcLibModules ghcFlavor = do
   copyFile mainFile rootModulePath
   lib <- mapM readCabalFile (cabalFileLibraries ghcFlavor)
   cabalPackageDb <- cabalPackageDb =<< ghcNumericVersion
-  let includeDirs = map ("-I" ++) (ghcLibIncludeDirs ghcFlavor)
+  let includeDirs = map ("-I" ++) (ghcLibIncludeDirs True ghcFlavor)
```

Comment:
Since ghcLibIncludeDirs ignores its first argument, we can drop this `True`

Aside: Seeing as ghcLibIncludeDirs is the same ghcLibParserIncludeDirs now, is there a reason we keep them as separate identifiers?

Reply:
yes, a formality where in that one can reasonably be expected to differ from the other and at some point probably will . expressing them as one is the other is a shortcut.